pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -364,17 +364,9 @@ func (as *availabilitySet) GetInstanceIDByNodeName(name string) (string, error)
 
 	machine, err = as.getVirtualMachine(types.NodeName(name))
 	if err != nil {
-		if as.CloudProviderBackoff {
```

Comment:
az.CloudProviderBackoff  and az.requestBackoff() is no longer checked for VM get, is it expected?

Reply:
With cache, we won't call ARM API frequently (GET and LIST), so backoff is not required for them. 

But notice that `CreateOrUpdate` is still using backoff.